src/components/autocomplete/demoBasicUsage/script.js

```diff
@@ -16,6 +16,12 @@
     self.selectedItemChange = selectedItemChange;
     self.searchTextChange   = searchTextChange;
 
+    self.newState = newState;
+
+    function newState(state) {
+      alert("Sorry! You'll need to create a Constituion for " + state + " first!");
```

Comment:
I was worried we might break the US by allowing people to create states.  Thankfully, there's an error. =p